PR For Host to Review Errors Discussed In Chat

shaniyaclement opened the pull request:
<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of an existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

---
**For errors in the chat**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

shaniyaclement committed:
Merge pull request #21 from shaniyaclement/main

Updating Shaniya's Commits to match Main

Commit:
Commit for host to review. Renders an error that gcp_conn_id doesn't exist bc it is not mocking properly and that .create is not called in reference to line 127. these should only get feedback and not be approved to merge

Commit:
.

Commit:
minor fix to params in mock request

shaniyaclement committed:
Merge pull request #26 from shaniyaclement/main

Updating Commits Branch to Match Main

Commit:
Minor removals based on Manav comments

Dippatel98 commented on tests/providers/google/cloud/hooks/test_datapipeline.py:
```diff
@@ -0,0 +1,128 @@
+    """
+    
+    def setup_method(self):
+        self.datapipeline_hook = DataPipelineHook(gcp_conn_id = "test_google_cloud_default")
```
Is there a reason the conn_id mentioned here `"test_google_cloud_default"` is different than on line 63 `"test_gcp_conn_id"`

shaniyaclement replied:
yes, I was just testing that the test worked without the mock by calling the actual connection before address the mock gcp conn error because I believe it'll take longer than writing the necessary test. once that is fixed, I will update them to match

Dippatel98 replied:
Makes sense

shaniyaclement replied:
Updated to use the actual connection id as suggested since the Dataflow example does the same

Commit:
test_create_data_pipeline was failing because "create()"
was not being called. this was because I used the wrong test_parent and my project id wasn't the same all around. fixed this and removed unnecessary patch. now focusing on mock gcp conn issue.

Dippatel98 commented:
Everything else is looking good. LGTM

Commit:
completed code. cleaned up unnecessary imports

shaniyaclement committed:
Merge branch 'main' into Shaniya's-Commits

shaniyaclement committed:
Merge branch 'main' into Shaniya's-Commits

Commit:
removed an overlooked conflict from when I merged the run test

blpham commented on airflow/providers/google/cloud/hooks/datapipeline.py:
```diff
@@ -56,14 +41,14 @@ def __init__(
-            gcp_conn_id=gcp_conn_id,
-            impersonation_chain=impersonation_chain,
+            gcp_conn_id = gcp_conn_id,
+            impersonation_chain = impersonation_chain,
```
I believe it's proper format to not have the spaces beside the '=' in keyword arguments
https://peps.python.org/pep-0008/#:~:text=PosInt%3A%20...-,Don%E2%80%99t,-use%20spaces%20around

shaniyaclement replied:
i fixed this for both of the hook and operators in each file

blpham commented on airflow/providers/google/cloud/hooks/datapipeline.py:
```diff
@@ -56,14 +41,14 @@ def __init__(
         """Returns a Google Cloud Data Pipelines service object."""
         http_authorized = self._authorize()
-        return build("datapipelines", "v1", http=http_authorized, cache_discovery=False)
+        return build("datapipelines", "v1", http = http_authorized, cache_discovery = False)
```
^ same here and everywhere else

shaniyaclement replied:
^

Commit:
formatting all files created for both operators and hooks

blpham reviewed (approved):
lgtm!!
